README.md

```
@@ -323,6 +323,65 @@ skills/
+curl -fsSL https://raw.githubusercontent.com/Enriquefft/openclaw-kapso-whatsapp/main/scripts/install.sh | bash
+```
+Downloads latest release, verifies SHA256 checksums, installs to ~/.local/bin.
+
+### Configuration
+| File | Purpose |
+|------|---------|
+| kapso-whatsapp-bridge.toml | Bridge configuration |
+| kapso-whatsapp-cli.toml | CLI configuration |
+
+### Delivery Modes
+| Mode | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)